src/styles/reset.css

```diff
@@ -0,0 +1,30 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: #060b35;
+  display: inline-block;
+}
+
+ul,
+ol,
+li {
+  list-style: none;
+}
```

Comment:
Use classes.
Also, we you don't have `ol` here
And `li` dosb't have list-style rule.

Reply:
this is common reset file. I thougth that its correct!